### PR TITLE
Release notes for 2019.2

### PR DESCRIPTION
**Pull request by sdrave**

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -4,6 +4,191 @@
...
+- `[#607] Replace sqlite caching <https://github.com/pymor/pymor/pull/607>`_ 
+
+
+
+Backward incompatible changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Dropped Python 3.5 support
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#790] Default to POD-Greedy for instationary problems <https://github.com/pymor/pymor/pull/790>`_
```
Comment (pmli) [inline docs/source/release_notes.rst]: Maybe a different category would be better?

**Comment by renefritze** [comment]
All(?) of the bullet list items seem to have dangling whitespace. Is that intentional, @sdrave ?

**Comment by sdrave** [comment]
@renefritze, no ..

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -4,6 +4,192 @@
...
+
+
+
+Backward incompatible changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Dropped Python 3.5 support
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#553] increase minimal python version to 3.6 <https://github.com/pymor/pymor/pull/553>`_
+- `[#584] Simply implementation of RuleTable`_
```
Comment (pmli) [inline docs/source/release_notes.rst]: When running `make docs` there is an `Unknown target name` for this line. Maybe an URL is missing?

**Comment by sdrave** [comment]
@pmli, can you write something regarding the sys-mor methods?

**Pull request merged**